Makefile

```diff
@@ -25,13 +25,14 @@ endef
 $(error $(NOT_IN_GOPATH_ERROR))
 endif
 
-CGO_CFLAGS=-I/$(JAVA_HOME)/include -I/$(JAVA_HOME)/include/darwin
-GOBIN=$(dir $(realpath $(firstword $(MAKEFILE_LIST))))build/bin
-GIT_COMMIT := $(shell git rev-parse --short HEAD)
+CGO_CFLAGS = -I/$(JAVA_HOME)/include -I/$(JAVA_HOME)/include/darwin
+GOBIN = $(dir $(realpath $(firstword $(MAKEFILE_LIST))))build/bin
+GIT_COMMIT = $(shell git describe --exact-match --tag 2>/dev/null || git rev-parse --short HEAD)
+AUTHOR = $(shell echo $$USER)
```

Comment:
Do we really need author? It's another thing to keep in mind, specify, fix bugs and test, and I'm not sure there is a benefit from it now.

Reply:
I received a few questions like "when and who did update the image on XYZ because it does not work?" recently so I though it might be useful. I defaulted it to env `$USER` so that no one needs to think about it when building an image. It's optional I would say.